1-setup/schema/simple/university-postgresql.sql

```diff
@@ -0,0 +1,48 @@
+-- SQL script to create DB, and tables. Then insert data in each table
+
+-- Run the DROP statment by itself
+DROP DATABASE IF EXISTS University;
+
+-- Create the new Database
+-- Run the CREATE statment by itself
+CREATE DATABASE University ENCODING 'UTF8';
+
+-- Select the new university database and open a new query
+
+SET SCHEMA 'public';
+
+-- Create Tables
```

Comment:
-- I have assumed all ids to be INTEGERS instead of VARCHAR 
, could we remove this?